// fxjs/Strict/index.js
"use strict";

require("core-js/modules/es.array.filter");

require("core-js/modules/es.array.flat");

require("core-js/modules/es.array.flat-map");

require("core-js/modules/es.array.iterator");

require("core-js/modules/es.array.map");

require("core-js/modules/es.array.sort");

require("core-js/modules/es.array.unscopables.flat");

require("core-js/modules/es.array.unscopables.flat-map");

require("core-js/modules/es.string.match");

require("core-js/modules/es.string.split");

var _addJs = require("./add.js");

var _appendJs = require('./append.js');

var _applyJs = require('./apply.js');

var _baseSelJs = require('./baseSel.js');

var _callJs = require('./call.js');

var _callsJs = require('./calls.js');

var _chunkJs = require('./chunk.js');

var _compactJs = require('./compact.js');

var _constantJs = require('./constant.js');

var _countByJs = require('./countBy.js');

var _curryJs = require('./curry.js');

var _curry2Js = require('./curry2.js');

var _debounceJs = require('./debounce.js');

var _deepFlatJs = require('./deepFlat.js');

var _defaultsJs = require('./defaults.js');

var _defaultToJs = require('./defaultTo.js');

var _delayJs = require('./delay.js');

var _differenceJs = require('./difference.js');

var _differenceByJs = require('./differenceBy.js');

var _differenceWithJs = require('./differenceWith.js');

var _dropJs = require('./drop.js');

var _dropRightJs = require('./dropRight.js');

var _dropUntilJs = require('./dropUntil.js');

var _dropWhileJs = require('./dropWhile.js');

var _eachJs = require('./each.js');

var _entriesJs = require('./entries.js');

var _everyJs = require('./every.js');

var _extendJs = require('./extend.js');

var _filterJs = require('./filter.js');

var _findJs = require('./find.js');

var _findWhereJs = require('./findWhere.js');

var _flatJs = require('./flat.js');

var _flatMapJs = require('./flatMap.js');

var _goJs = require('./go.js');

var _go1Js = require('./go1.js');

var _goSJs = require('./goS.js');

var _groupByJs = require('./groupBy.js');

var _hasJs = require('./has.js');

var _headJs = require('./head.js');

var _hiJs = require('./hi.js');

var _htmlJs = require('./html.js');

var _identityJs = require('./identity.js');

var _indexByJs = require('./indexBy.js');

var _initialJs = require('./initial.js');

var _intersectionJs = require('./intersection.js');

var _intersectionByJs = require('./intersectionBy.js');

var _intersectionWithJs = require('./intersectionWith.js');

var _isArrayJs = require('./isArray.js');

var _isFunctionJs = require('./isFunction.js');

var _isIterableJs = require('./isIterable.js');

var _isMatchJs = require('./isMatch.js');

var _isStopJs = require('./isStop.js');

var _isStringJs = require('./isString.js');

var _isUndefinedJs = require('./isUndefined.js');

var _joinJs = require('./join.js');

var _keysJs = require('./keys.js');

var _lastJs = require('./last.js');

var _logJs = require('./log.js');

var _mapJs = require('./map.js');

var _mapEntriesJs = require('./mapEntries.js');

var _mapObjectJs = require('./mapObject.js');

var _matchJs = require('./match.js');

var _maxJs = require('./max.js');

var _maxByJs = require('./maxBy.js');

var _minJs = require('./min.js');

var _minByJs = require('./minBy.js');

var _negateJs = require('./negate.js');

var _noopJs = require('./noop.js');

var _nopJs = require('./nop.js');

var _notJs = require('./not.js');

var _objectJs = require('./object.js');

var _omitJs = require('./omit.js');

var _partitionJs = require('./partition.js');

var _pickJs = require('./pick.js');

var _pipeJs = require('./pipe.js');

var _pipe1Js = require('./pipe1.js');

var _pipeSJs = require('./pipeS.js');

var _pluckJs = require('./pluck.js');

var _prependJs = require('./prepend.js');

var _promiseAllEntriesJs = require('./promiseAllEntries.js');

var _promiseAllObjectJs = require('./promiseAllObject.js');

var _rangeJs = require('./range.js');

var _reduceJs = require('./reduce.js');

var _reduceSJs = require('./reduceS.js');

var _rejectJs = require('./reject.js');

var _selJs = require('./sel.js');

var _someJs = require('./some.js');

var _sortJs = require('./sort.js');

var _sortByJs = require('./sortBy.js');

var _sortByDescJs = require('./sortByDesc.js');

var _sortDescJs = require('./sortDesc.js');

var _splitJs = require('./split.js');

var _splitEveryJs = require('./splitEvery.js');

var _stopJs = require('./stop.js');

var _stopIfJs = require('./stopIf.js');

var _stringJs = require('./string.js');

var _strMapJs = require('./strMap.js');

var _sumJs = require('./sum.js');

var _sumByJs = require('./sumBy.js');

var _tailJs = require('./tail.js');

var _takeJs = require('./take.js');

var _take1Js = require('./take1.js');

var _takeAllJs = require('./takeAll.js');

var _takeUntilJs = require('./takeUntil.js');

var _takeWhileJs = require('./takeWhile.js');

var _tapJs = require('./tap.js');

var _throttleJs = require('./throttle.js');

var _toIterJs = require('./toIter.js');

var _unionJs = require('./union.js');

var _unionByJs = require('./unionBy.js');

var _uniqueJs = require('./unique.js');

var _uniqueByJs = require('./uniqueBy.js');

var _unzipJs = require('./unzip.js');

var _valuesJs = require('./values.js');

var _zipJs = require('./zip.js');

var _zipObjJs = require('./zipObj.js');

var _zipWithJs = require('./zipWith.js');

exports.add = _addJs;
exports.append = _appendJs;
exports.apply = _applyJs;
exports.baseSel = _baseSelJs;
exports.base_sel = _baseSelJs;
exports.call = _callJs;
exports.calls = _callsJs;
exports.chunk = _chunkJs;
exports.compact = _compactJs;
exports.constant = _constantJs;
exports.countBy = _countByJs;
exports.count_by = _countByJs;
exports.curry = _curryJs;
exports.curry2 = _curry2Js;
exports.debounce = _debounceJs;
exports.deepFlat = _deepFlatJs;
exports.deepFlatten = _deepFlatJs;
exports.deep_flat = _deepFlatJs;
exports.deep_flatten = _deepFlatJs;
exports.defaults = _defaultsJs;
exports.defaultTo = _defaultToJs;
exports.default_to = _defaultToJs;
exports.delay = _delayJs;
exports.difference = _differenceJs;
exports.differenceBy = _differenceByJs;
exports.difference_by = _differenceByJs;
exports.differenceWith = _differenceWithJs;
exports.difference_with = _differenceWithJs;
exports.drop = _dropJs;
exports.dropRight = _dropRightJs;
exports.drop_right = _dropRightJs;
exports.dropUntil = _dropUntilJs;
exports.drop_until = _dropUntilJs;
exports.dropWhile = _dropWhileJs;
exports.drop_while = _dropWhileJs;
exports.each = _eachJs;
exports.forEach = _eachJs;
exports.entries = _entriesJs;
exports.every = _everyJs;
exports.extend = _extendJs;
exports.filter = _filterJs;
exports.find = _findJs; //ok

exports.findWhere = _findWhereJs;
exports.find_where = _findWhereJs;
exports.flat = _flatJs;
exports.flatten = _flatJs;
exports.flatMap = _flatMapJs;
exports.flat_map = _flatMapJs;
exports.go = _goJs;
exports.go1 = _go1Js;
exports.goS = _goSJs;
exports.groupBy = _groupByJs;
exports.group_by = _groupByJs;
exports.has = _hasJs;
exports.head = _headJs;
exports.hi = _hiJs;
exports.html = _htmlJs;
exports.identity = _identityJs;
exports.indexBy = _indexByJs;
exports.index_by = _indexByJs;
exports.initial = _initialJs;
exports.intersection = _intersectionJs;
exports.intersectionBy = _intersectionByJs;
exports.intersection_by = _intersectionByJs;
exports.intersectionWith = _intersectionWithJs;
exports.intersection_with = _intersectionWithJs;
exports.isArray = _isArrayJs;
exports.is_array = _isArrayJs;
exports.isFunction = _isFunctionJs;
exports.is_function = _isFunctionJs;
exports.isIterable = _isIterableJs;
exports.is_iterable = _isIterableJs;
exports.isMatch = _isMatchJs;
exports.is_match = _isMatchJs;
exports.isStop = _isStopJs;
exports.is_stop = _isStopJs;
exports.isString = _isStringJs;
exports.is_string = _isStringJs;
exports.isUndefined = _isUndefinedJs;
exports.is_undefined = _isUndefinedJs;
exports.join = _joinJs;
exports.keys = _keysJs;
exports.last = _lastJs;
exports.log = _logJs;
exports.map = _mapJs;
exports.mapEntries = _mapEntriesJs;
exports.map_entries = _mapEntriesJs;
exports.entriesMap = _mapEntriesJs;
exports.entries_map = _mapEntriesJs;
exports.mapObject = _mapObjectJs;
exports.map_object = _mapObjectJs;
exports.match = _matchJs;
exports.max = _maxJs;
exports.maxBy = _maxByJs;
exports.max_by = _maxByJs;
exports.min = _minJs;
exports.minBy = _minByJs;
exports.min_by = _minByJs;
exports.negate = _negateJs;
exports.noop = _noopJs;
exports.nop = _nopJs;
exports.not = _notJs;
exports.object = _objectJs;
exports.fromEntries = _objectJs;
exports.from_entries = _objectJs;
exports.omit = _omitJs;
exports.partition = _partitionJs;
exports.pick = _pickJs;
exports.pipe = _pipeJs;
exports.pipe1 = _pipe1Js;
exports.pipeS = _pipeSJs;
exports.pluck = _pluckJs;
exports.prepend = _prependJs;
exports.promiseAllEntries = _promiseAllEntriesJs;
exports.promise_all_entries = _promiseAllEntriesJs;
exports.promiseAllObject = _promiseAllObjectJs;
exports.promise_all_object = _promiseAllObjectJs;
exports.range = _rangeJs;
exports.reduce = _reduceJs;
exports.reduceS = _reduceSJs;
exports.reject = _rejectJs;
exports.sel = _selJs;
exports.some = _someJs;
exports.sort = _sortJs;
exports.sortBy = _sortByJs;
exports.sort_by = _sortByJs;
exports.sortByDesc = _sortByDescJs;
exports.sort_by_desc = _sortByDescJs;
exports.sortDesc = _sortDescJs;
exports.sort_desc = _sortDescJs;
exports.split = _splitJs;
exports.splitEvery = _splitEveryJs;
exports.split_every = _splitEveryJs;
exports.stop = _stopJs;
exports.stopIf = _stopIfJs;
exports.stop_if = _stopIfJs;
exports.string = _stringJs;
exports.strMap = _strMapJs;
exports.str_map = _strMapJs;
exports.scat = _strMapJs;
exports.sum = _sumJs;
exports.sumBy = _sumByJs;
exports.sum_by = _sumByJs;
exports.tail = _tailJs;
exports.rest = _tailJs;
exports.take = _takeJs;
exports.take1 = _take1Js;
exports.takeAll = _takeAllJs;
exports.take_all = _takeAllJs;
exports.takeUntil = _takeUntilJs;
exports.take_until = _takeUntilJs;
exports.takeWhile = _takeWhileJs;
exports.take_while = _takeWhileJs;
exports.tap = _tapJs;
exports.throttle = _throttleJs;
exports.toIter = _toIterJs;
exports.to_iter = _toIterJs;
exports.union = _unionJs;
exports.unionBy = _unionByJs;
exports.union_by = _unionByJs;
exports.unique = _uniqueJs;
exports.uniq = _uniqueJs;
exports.uniqueBy = _uniqueByJs;
exports.unique_by = _uniqueByJs;
exports.unzip = _unzipJs;
exports.values = _valuesJs;
exports.zip = _zipJs;
exports.zipObj = _zipObjJs;
exports.zip_obj = _zipObjJs;
exports.zipWith = _zipWithJs;
exports.zip_with = _zipWithJs;
